vmm/src/device_manager.rs

```diff
@@ -1306,8 +1310,8 @@ impl DeviceManager {
                 ))
             };
             self.add_migratable_device(
-                Arc::clone(&virtio_net_device) as Arc<Mutex<dyn Migratable>>
-            );
+                            Arc::clone(&virtio_net_device) as Arc<Mutex<dyn Migratable>>
+                        );
```

Comment:
Weird whitespace change! (has cargo fmt changed the way it works...?)

Reply:
Oh yeah that's weird, I didn't spot that, let me check.

Reply:
I changed it :)